packages/lesswrong/components/questions/AnswersSorting.tsx

```diff
@@ -6,6 +6,7 @@ import * as _ from 'underscore';
 import type { Option } from '../common/InlineSelect';
 import { isFriendlyUI } from '../../themes/forumTheme';
 
+// TODO: use postViewOptions
```

Comment:
Is this something you want to do before merging?

Reply:
Upon looking at this, commentViewOptions seemed the right one to use, with answers being more like comments than they are like posts. I've made the change, which just required a re-ordering of the default comment options. I hope what I did makes sense.